controllers/get-time.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const moment = require('moment-timezone')
+
+const controller = {
+  execute: ( param ) => {
+
+    if(!param || typeof(param) !== 'number') {
```

Comment:
The moment-timezone module handles values outside those of 'valid timezones' rather elegantly. It does not enforce any strict relationship between the offset and any understanding of a timezone; it is quite simply a chronological offset as represented by a number. So if the offset value is 100 or 10,000 then it will be calculated as that many minutes from the current time - irrespective of any preeconcieved notions of timezones.

I did not understand the assignment instructions to mean that the offset was to be one of a timezone, but rather of a value which *could* be a timezone, but not exclusively. If it is to be strictly one of timezones then strict validation tests could be applied which: 

1. force values to the closest legitimate timezone offset
2. return a default time value, ignoring the offset
3. return an error
4. some combination of the above

It really depends on the use-case.

A simplistic validation check could check if the number was in the range -12 to +12; if outside that range it could return a JSON object which contained the current time, with the offset at 0, and additionally an error object which (depending upon the given standard) detailed the error as being that the offset is out of bounds.

* the module actually does to a check for absolute values less than 16 (peculiarly), but only uses this to convert the number from a double digit whole number to one which represents minutes, by multiplying by 60.

https://github.com/moment/moment/blob/5c546134b85ce731ba420e77779d616586200e2b/src/lib/units/offset.js

Reply:
@seannicholls great explanation, thanks :)